TLM/TLM/UI/SubTools/LaneArrowTool.cs

```diff
@@ -8,9 +8,15 @@ namespace TrafficManager.UI.SubTools {
     using TrafficManager.Manager.Impl;
```

Comment:
Please wait until my LaneArrows branch is merged, GUI interaction and keyboard handling is different there